app/locales/zh-TW/gridlocale.js

```diff
@@ -0,0 +1,269 @@
+// Example locale file for English, give this to your locale team to translate
+
+const AG_GRID_LOCALE_ZH_TW = {
```

Comment:
I wonder if there are predefined locale files.. seems like a waste to have every team translate their own file